src/main.js

```diff
@@ -92,6 +92,30 @@ function displayNewRomCom() {
 function saveRomCom() {
   if (!savedRomComs.includes(currentRomCom)) {
     savedRomComs.push(currentRomCom);
+    displaySavedRomComs();
+  }
+};
+
+function displaySavedRomComs() {
+  var renderer = '';
+  for (var i = 0; i < savedRomComs.length; i++) {
+    renderer += `<section ondblclick='deleteRomComs(this.id)' class= "mini-cover" id="${savedRomComs[i].id}">
```

Comment:
Great use of using this key word to access values in an object through HTML - nifty!!